tests/wicked/before_test.pm

```diff
@@ -41,6 +37,10 @@ sub run {
         $self->setup_static_network($self->get_ip(is_wicked_ref => check_var('IS_WICKED_REF', 1), type => 'host'));
     }
     $self->get_from_data('wicked/ifbind.sh', '/bin/ifbind.sh', executable => 1);
+    record_info('INFO', 'Checking that network service is up');
+    systemctl('is-active network');
+    systemctl('is-active wicked');
+    zypper_call('--quiet in openvpn', timeout => 200) if (check_var('WICKED', 'advanced'));
```

Comment:
please move this line just after line 41/37. It relevant only for WICKED=advanced

Reply:
The condition for WICKED=advanced is there. If it was moved, it would be before is-active tests and logic would not be clear (like using network for installation before actual network check).